diff --git a/node_modules/cornerstone-tools/dist/cornerstoneTools.js b/node_modules/cornerstone-tools/dist/cornerstoneTools.js
index 384f909..9ecfdf6 100644
--- a/node_modules/cornerstone-tools/dist/cornerstoneTools.js
+++ b/node_modules/cornerstone-tools/dist/cornerstoneTools.js
@@ -4955,6 +4955,7 @@ function brushTool(brushToolInterface) {
 
     var configuration = brushTool.getConfiguration();
     var colormapId = configuration.colormapId;
+    console.log('*****************************colormap*************************')
 
     if (!colormapId) {
       colormapId = 'BrushColorMap';
